.travis.yml

```diff
@@ -10,14 +10,17 @@ jobs:
     - stage:
       python: 3.6
       env: TOXENV=py36
+    - stage:
+      python: 3.7
+      env: TOXENV=py37
```

Comment:
In order to get this to work in Travis you need to set the distro. See pytest-selenium for an example:

```yml
    - stage:
      python: 3.7
      dist: xenial
      sudo: required
```